### PR TITLE
[sailfish-office] Add share context menu on main page. Fixes JB#41808

### PR DESCRIPTION
**Pull request by Nrsisti**
… JB#41808

### CODE REVIEW

File: qml/FileListPage.qml [hunk 1]
```diff
@@ -241,6 +241,13 @@ Page {
                             listItem.deleteFile()
                         }
                     }
+                    MenuItem {
+                        //: Share a file
+                        //% "Share"
+                        text: qsTrId("sailfish-office-me-share")
+                        onClicked: pageStack.push("Sailfish.TransferEngine.SharePage",
+                                                   { serviceFilter: ["e-mail"], mimeType: model.fileMimeType, source: model.filePath })
+                    }
```
Comment (rainemak) [inline qml/FileListPage.qml]: Use the same order as in Settings->Storage so "Share" first and "Delete" 2nd context menu item.

Comment (Nrsisti) [reply qml/FileListPage.qml]: Updated

File: qml/FileListPage.qml [hunk 2]
```diff
@@ -233,6 +233,13 @@ Page {
             menu: Component {
                 ContextMenu {
                     id: contextMenu
+                    MenuItem {
+                        //: Share a file
+                        //% "Share"
+                        text: qsTrId("sailfish-office-me-share")
```
[the remark this answers is not in the document]
Comment (Nrsisti) [reply qml/FileListPage.qml]: Updated

**Comment by rainemak** [comment]
Change "Contributes to JB#41808" to "Fixes JB#41808"

**Pull request merged**